packages/Adapters/POD_C/test/tstC_API.cpp

```diff
@@ -0,0 +1,92 @@
+#include "DTK_C_API.h"
+
+#include "Teuchos_UnitTestHarness.hpp"
+#include "Teuchos_DefaultComm.hpp"
+
+#include "mpi.h"
+
+namespace {
+
+TEUCHOS_UNIT_TEST( C_API, create_apply_delete_map )
+{
+  // get communicator
+  Teuchos::RCP<const Teuchos::Comm<int>> comm =
+    Teuchos::DefaultComm<int>::getComm();
+
+  // let us use INFO format
+  std::string options;
+  options = "\n"\
+    "\"Map Type\" \"Spline Interpolation\"\n"\
+    "\"Basis Type\" \"Wendland\"\n"\
+    "\"Basis Order\" 0\n"\
+    "\"Spatial Dimension\" 3\n"\
+    "\"Target Coord Data Layout\" \"Contiguous\"\n"\
+    "\"Source Coord Data Layout\" \"Contiguous\"\n"\
```

Comment:
Why do dimension and layout need to be in options if they are input parameters in the API?